### ISSUE TITLE
Textboxes are not the same size in the Mirrors section of Cluster Configuration

### ISSUE THREAD

**Issue by deniseschannon** [rancher/dashboard#3606]
When creating a RKE2 cluster in the "cluster configuration", the registries section has textboxes that don't have the same height in the "Mirrors". 

<img width="702" alt="Screen Shot 2021-08-02 at 3 41 51 PM" src="https://user-images.githubusercontent.com/11410997/127932665-9f2202ce-ea53-4ade-bc4c-a70377aad156.png">

**Comment by deniseschannon** [comment]
This only happens if I have a small window and add a key value pair so that the helper text takes up 2 lines inside the text box. I'm moving this out of v2.6 as it's not critical and a small use case. If you expand your window and add a row, it adjusts to match.

**Comment by brudnak** [comment]
**My checks PASSED**

**TLDR:**

> Original issue has been corrected. But the fix has introduced a new issue. Linked at the bottom of this comment. And here: [https://github.com/rancher/dashboard/issues/4371](https://github.com/rancher/dashboard/issues/4371)
  
**Reproduction Environment:**

| Tech                 | Version                                 |
| -------------------- | --------------------------------------- |
| Rancher version      | v2.6.0                                  |
| Rancher cluster type | Docker install                          |
| Docker version       | 20.10.7, build 20.10.7-0ubuntu1~21.04.2 |
| Helm version         | v2.16.8-rancher1                        |
| Machine              | v0.15.0-rancher69                       |
| Browser type         | Google Chrome                           |
| Browser version      | 94.0.4606.81 (Official Build) (x86_64)  |

> API Token Created for Digital Ocean
 
**Reproduction steps:**

1. Starting from Rancher home screen `/dashboard/home`
2. Click hamburger menu -> Cluster Management -> Create
3. Toggle RKE1 to RKE2/k3s
4. Click DigitalOcean under `Provision new nodes and create a cluster using RKE2/K3s (Tech Preview)` -> Registries
5. From `Registries` select the `Configure advanced containerd mirroring and registry authentication options` radio button
6. Click `Add Mirror`
7. Decrease the width of the browser until it's about `1088`
    - This can be checked by running the following form the Google Chrome console
    - `console.log(window.innerWidth);`
8. Now the UI should look like screen shot labeled `A1`
9. Type some text that is multiline in the `Mirror Endpoints` textarea. Here is an example
```txt
a.registry.com:5000,
b.registry.com:5000,
c.registry.com:5000,
d.registry.com:5000,
```
10. Once the text has been entered. Delete it
11. Observe the height of the `Mirror Endpoints` text area, both at the current size and even if you increase the width past 1088. See screen shot `A2` for an example of the issue being reproduced

**Screen Shots**

<img width="888" alt="rancher-dashboard-a1" src="https://user-images.githubusercontent.com/40877149/137226239-0a839098-d438-47aa-b647-e5ffdf1633fc.png">

<img width="1069" alt="rancher-dashboard-a2" src="https://user-images.githubusercontent.com/40877149/137226273-73cb5e1a-b098-4fb3-b8db-f8fff82047a3.png">

-----
  
**Validation Environment:**

| Tech                 | Version                                            |
| -------------------- | -------------------------------------------------- |
| Rancher version      | v2.6-9801a059cf239934f5093ba8e3d15bd3fd9460d5-head |
| Rancher cluster type | Docker install                                     |
| Docker version       | 19.03.15, build 99e3ed8919                         |
| Helm version         | v2.16.8-rancher1                                   |
| Machine              | v0.15.0-rancher70                                  |
| Browser type         | Google Chrome                                      |
| Browser version      | 94.0.4606.81 (Official Build) (x86_64)             |
  

  **Validation steps:**

> The same steps 1 through 11 as listed above. However, now the height of the text area adjusts back to it's original size as expected.

The original issue has been corrected.

While testing a new issue was discovered that is related to this fix and can be found here: [https://github.com/rancher/dashboard/issues/4371](https://github.com/rancher/dashboard/issues/4371)
